.babelrc

```diff
@@ -2,43 +2,54 @@
   "env": {
     "test": {
       "presets": [
-        ["env", {
-          "targets": {
-            "node": "current"
-          },
-        }],
-        "react",
+        [
+          "@babel/preset-env",
+          {
+            "targets": {
+              "node": "current"
+            }
+          }
+        ],
       ],
-      plugins: [
-        "transform-class-properties",
-        "transform-flow-strip-types",
-        "syntax-dynamic-import",
-        ["transform-object-rest-spread", { "useBuiltIns": true }],
+      "plugins": [
         "istanbul"
-      ],
+      ]
     }
   },
-  presets: [
-    ["env", {
-      "targets": {
-        "browsers": [
-          "last 3 Chrome versions",
-          "last 3 Firefox versions",
-          "last 2 Edge versions",
-          "last 1 Safari versions",
-          "last 1 iOS versions"
-        ]
-      },
-      "modules": false,
-    }],
-    "react",
-    /*"react-optimize",*/
-  ],
-  plugins: [
-    "transform-class-properties",
-    "transform-flow-strip-types",
-    ["import", { libraryName: "antd" }],
-    "syntax-dynamic-import",
-    ["transform-object-rest-spread", { "useBuiltIns": true }]
+  "presets": [
+    [
+      "@babel/preset-env",
+      {
+        "targets": {
+          "browsers": [
+            "last 3 Chrome versions",
+            "last 3 Firefox versions",
+            "last 2 Edge versions",
+            "last 1 Safari versions",
+            "last 1 iOS versions"
+          ]
+        },
+        "modules": false
+      }
+    ],
+    "@babel/preset-react",
+    "@babel/preset-flow"
   ],
+  "plugins": [
+    "@babel/plugin-proposal-class-properties",
+    // Currently broken (as of 1.9.1) - does not rename the import (_select) in a line like `const { Option } = Select;`
```

Comment:
Is there an issue we can track? This will blow up the bundle size a bit, since there won't be any tree shaking for antd, right?

Reply:
I couldn't find an issue in the babel-plugin-import repo. I've tried to repro on codesandbox, but the plugin doesn't seem to be applied there (https://codesandbox.io/s/8k17q911y2), didn't find out why yet. Nevertheless, this will only affect the tests, so I think there wasn't even any tree shaking to begin with.